docs/egg_model_example.rst

```diff
@@ -0,0 +1,149 @@
+
+.. image:: ./_static/sintef_logo.jpg
+   :scale: 10%
+
+
+Data files
+==========
+
+- `The Egg Model - data files  <https://data.4tu.nl/articles/The_Egg_Model_-_data_files/12707642>`_ [2]
+
+
+Modifications to the simulations setup
+======================================
+
+We have modified the Egg model input file `Egg_Model_ECL.DATA` in order to follows Flownet workflow.
+
+1. COMPAT section is changed to have differences in depth between injectors and producers. With this, we avoid problems related to degenerate tetrahedron that we obtain in the triangulation procedure for the network generation.
+
+::
+
+        COMPDAT
+            'INJECT1'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /                    
+            'INJECT2'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /            
+            'INJECT3'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /
+            'INJECT4'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /
+            'INJECT5'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /
+            'INJECT6'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /
+            'INJECT7'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /
+            'INJECT8'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /
+            'PROD1'      2*    1     3 'OPEN' 2*     0.2 	1*          0 / 
+            'PROD2'      2*    1     3 'OPEN' 2*     0.2 	1*          0 / 
+            'PROD3'      2*    1     3 'OPEN' 2*     0.2 	1*          0 / 
+            'PROD4'      2*    1     3 'OPEN' 2*     0.2	1*          0 /
+        /
+        
+        
+
+2. The section PROPS is saved as an independent file `PROPS.inc`.  
+
+::
+
+            DENSITY
+                900 1000          1 /
+            PVCDO
+                400          1 1.000E-05        5        0/
+            PVTW
+                400          1 1.000E-05        1        0/
+
+                
+3. Output variables of interest have been specified in the SUMMARY section.
+
+::
+
+        SUMMARY
+            FOPR
+            FWPR
+            FWIR
+            WOPR
+            'PROD1'
+            'PROD2'
+            'PROD3'
+            'PROD4'
+            /
+            WWPR
+            'PROD1'
+            'PROD2'
+            'PROD3'
+            'PROD4'
+            /
+            WWIR
+            'INJECT1' 
+            'INJECT2'
+            'INJECT3' 
+            'INJECT4' 
+            'INJECT5'
+            'INJECT6'
+            'INJECT7'
+            'INJECT8'
+            /   
+            FOPT
+            FWPT
+            FWIT
+            WLPR
+            'PROD1'
+            'PROD2'
+            'PROD3'
+            'PROD4'
+            /
+            WBHPH
+            /
+            WBHP
+            /
+
+
+4. Then you can run OPM FLOW to update input files to FlowNet
+
+.. code-block:: bash
+
+    flow /egg_model/input_file/Egg_Model_ECL.DATA
```

Comment:
I'm always surprised that this model was released with a mix of lower case and upper case letters - Eclipse (on Unix) can't handle this. Did you ran into problems with Flow?

```suggestion
    flow Egg_Model_ECL.DATA
```

Reply:
I got no problems with Flow, but I haven't tried with Eclipse